src/wr_hsi_sensing/package.xml

```diff
@@ -37,19 +37,27 @@
   <!--   <build_depend>roscpp</build_depend> -->
   <!--   <exec_depend>roscpp</exec_depend> -->
     <depend>rospy</depend>
+    <depend>ublox_gps</depend>
+    <depend>mapviz</depend>
```

Comment:
It might be better to have ublox_gps and mapviz as `exec_depend`, since I don't think they are needed at build time.